bundle/downstream/bundle.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM scratch
+
+LABEL com.redhat.delivery.operator.bundle=true
+LABEL com.redhat.delivery.openshift.ocp.versions="v4.7-v4.8"
+LABEL com.redhat.openshift.versions="v4.7-v4.9"
```

Comment:
Why are we limiting this to 4.9 instead of just saying "anything higher than 4.7" ? This can be achieved by just changing this to:
```
LABEL com.redhat.openshift.versions=v4.7
```

Reply:
Oh, I have just seen your conversation above with Brett, I am resolving this comment.